gettsim/benefits/elterngeld.py

```diff
@@ -172,47 +175,47 @@ def check_eligibilities(household, params):
     """
     household["elternzeit_anspruch"] = False
     household["geschw_bonus"] = False
-    household["num_mehrlinge"] = 0
+    household["anz_mehrlinge"] = 0
```

Comment:
Can we just use something that counts kids born on one date rather than subtracting 1? I find this very confusing and it is impossible for a non-German speaker to find out what we mean.

Reply:
It is really hard for me to think of the right variable. I mean, if a non-english speaker translates mehrlinge, he/she will exactly get multiples and that's what we count. I would add a comment, which clarifies the notion?

Reply:
But when thinking of multiples, twins would get a value of 2, triplets of 3, ... IIUC, this would count twins as anz_mehrlinge=1 ?

Reply:
True. But if I change the counting, a single child gets anz_mehrlinge=1 and it is not a mehrling. I found that confusing as well. I now changed to a bonus variable. Maybe that works!